releasenotes/notes/deprecate-fake-backends-4dd275cf9a30d41f.yaml

```
@@ -0,0 +1,50 @@
+---
+deprecations:
+  - |
+    The core functionality of the :mod:`qiskit.providers.fake_provider` module has been migrated to
+    the new ``qiskit_ibm_runtime.fake_provider`` module. For this reason, the following elements in
+    the :mod:`qiskit.providers.fake_provider` have been deprecated as of Qiskit 0.46 and will be
+    removed in Qiskit 1.0:
```

```suggestion
    The :mod:`qiskit.providers.fake_provider` module has been migrated to
    the ``qiskit-ibm-runtime`` Python package. For this reason, the following elements in
    the :mod:`qiskit.providers.fake_provider` have been deprecated as of Qiskit 0.46 and will be
    removed in Qiskit 1.0:
```

Maybe something like this is a bit clearer?